src/xo-app/host/tab-patches.js

```diff
@@ -171,8 +169,6 @@ export default class HostPatches extends Component {
             {isEmpty(missingPatches)
               ? <TabButton
                 disabled
-                handler={this._installAllPatchesWarning}
-                handlerParam={installAllPatches}
```

Comment:
Check the browser console for warnings.

We should aim at 0 warnings (even if that's not the case for now because of `react-bootstrap`).